framework/addons/utils.py

```diff
@@ -53,7 +53,15 @@ def read_capabilities(filename):
 here = os.path.abspath(here)
 
 lookup = TemplateLookup(
-    directories=[os.path.join(here, 'templates')]
+    directories=[os.path.join(here, 'templates')],
+    default_filters=[
+        'unicode',  # default filter; must set explicitly when overriding
+        'temp_ampersand_fixer',  # FIXME: Temporary workaround for data stored in wrong format in DB. Unescape it before it gets re-escaped by Markupsafe.
```

Comment:
I have added this to the checklist for JIRA issue 4432. When that PR lands, the filter should become unnecessary.

https://openscience.atlassian.net/browse/OSF-4432